products/db-checkbook/build_scripts/summarization.py

```diff
@@ -0,0 +1,171 @@
+from typing import Union
+import pandas as pd
+import logging
+
+from . import (
+    OUTPUT_DIR,
+    BUILD_OUTPUT_FILENAME,
+    SUMMARY_STATS_DESCRIBE_FILENAME,
+    SUMMARY_STATS_LOG_FILENAME,
+)
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(message)s",
+    handlers=[
+        logging.FileHandler(OUTPUT_DIR / SUMMARY_STATS_LOG_FILENAME),
+        logging.StreamHandler(),
+    ],
+)
+
+_percent_of_total_records = lambda selected_data, all_data: round(
```

Comment:
Any reason to use lambdas here instead of def? [FYI](https://www.flake8rules.com/rules/E731.html

Reply:
I'm not sure I'd want to enforce that yet before discussing the various rules. I do think it would be beneficial for folks to enable a PEP8 linter in their editors though!